addons/l10n_it_edi/models/account_edi_proxy_user.py

```diff
@@ -12,6 +12,7 @@ class AccountEdiProxyClientUser(models.Model):
     _inherit = 'account_edi_proxy_client.user'
 
     proxy_type = fields.Selection(selection_add=[('l10n_it_edi', 'Italian EDI')], ondelete={'l10n_it_edi': 'cascade'})
+    company_id = fields.Many2one('res.company', string='Company', required=True, default=lambda self: self.env.company.root_id)
```

Comment:
You can't add a new column in stable

Reply:
Ok, let's place this...
```py
    def _register_proxy_user(self, company, proxy_type, edi_mode):
        if proxy_type == 'l10n_it_edi':
            company = company.root_id
        return super._register_proxy_user(company, proxy_type, edi_mode)
```
... in `l10n_it_edi/models/account_edi_proxy_user.py`